.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ workflows:
                 - develop
       - pypi_publish:
           requires:
-            - build_test
+            - test_pypi_publish

```

Comment:
why did you change this?

Reply:
updating the requirement for publish_pypi job